HA/xcatha.py

```diff
@@ -56,7 +56,7 @@
 #configure logger
 LOG_FORMAT = "%(asctime)s - %(levelname)s - %(message)s"
 DATE_FORMAT = "%m/%d/%Y %H:%M:%S %p"
-logging.basicConfig(filename = os.path.join(os.getcwd(), 'xcatha.log'), level = logging.INFO, filemode = 'a', format = LOG_FORMAT, datefmt=DATE_FORMAT)
+logging.basicConfig(filename = os.path.join(os.getcwd(), 'xcatha.log'), level = logging.DEBUG, filemode = 'a', format = LOG_FORMAT, datefmt=DATE_FORMAT)
```

Comment:
I think this value should be an option to this script so that the end user can more easily change the logging value on each run of the script. Defaulting to "debug" may be too intense.

Reply:
Hi @whowutwut,  current we do not have ``--debug`` option,  so all logger output will print into log file and in output. And "dryrun" output is similar with the normal output. If xcatha.py only print some key stage message, some steps need time to wait, it looks like the script hang, so now it print every command in output, user know that xcatha.py continue working.